packages/base-zone/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@agoric/base-zone",
+  "version": "1.0.0",
+  "description": "Allocation zone abstraction library and heap implementation",
+  "type": "module",
+  "repository": "https://github.com/Agoric/agoric-sdk",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "exit 0",
+    "test": "ava",
+    "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
+    "test:xs": "exit 0",
+    "lint-fix": "yarn lint:eslint --fix",
+    "lint": "run-s --continue-on-error lint:*",
+    "lint:types": "tsc -p jsconfig.json",
+    "lint:eslint": "eslint ."
+  },
+  "exports": {
+    ".": "./src/index.js",
+    "./heap.js": "./heap.js",
+    "./tools/*": "./tools/*"
+  },
+  "keywords": [],
+  "author": "Agoric",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@agoric/store": "^0.9.2",
+    "@endo/exo": "^0.2.3",
+    "@endo/far": "^0.2.19",
+    "@endo/pass-style": "^0.1.4",
+    "@endo/patterns": "^0.2.3"
+  },
+  "devDependencies": {
```

Comment:
This will still have a cycle. Maybe the base-zone tests that need internal live there or in zone?

Reply:
I chased down all the cycles from `@agoric/internal`.  Lerna now just complains about a single nested cycle (deploy-script-support -> vats).

Reply:
I'm seeing four cycles left:
```
@agoric/deploy-script-support -> @agoric/vats -> @agoric/inter-protocol -> @agoric/deploy-script-support
@agoric/governance -> (nested cycle: @agoric/deploy-script-support -> @agoric/vats -> @agoric/inter-protocol -> @agoric/deploy-script-support) -> @agoric/governance
@agoric/smart-wallet -> (nested cycle: @agoric/governance -> (nested cycle: @agoric/deploy-script-support -> @agoric/vats -> @agoric/inter-protocol -> @agoric/deploy-script-support) -> @agoric/governance) -> @agoric/smart-wallet
(nested cycle: @agoric/smart-wallet -> (nested cycle: @agoric/governance -> (nested cycle: @agoric/deploy-script-support -> @agoric/vats -> @agoric/inter-protocol -> @agoric/deploy-script-support) -> @agoric/governance) -> @agoric/smart-wallet) -> (nested cycle: @agoric/smart-wallet -> (nested cycle: @agoric/governance -> (nested cycle: @agoric/deploy-script-support -> @agoric/vats -> @agoric/inter-protocol -> @agoric/deploy-script-support) -> @agoric/governance) -> @agoric/smart-wallet)
```